tools/mcsema_disass/ida/collect_variable.py

```diff
@@ -365,6 +365,11 @@ def _translate_reg_64(reg):
 _base_ptr_format = "[{}+".format(_base_ptr)
 _stack_ptr_format = "[{}+".format(_stack_ptr)
 
+def floor_key(d, key):
+  L1 = list(k for k in d if k <= key)
```

Comment:
Can `k` ever be zero, and if so, the code below doesn't handle the case, because it tends to do `if start_:`, whereas what you probably want is `if start_ is not None:`.